ecs-cli/integ/runner.go

```diff
@@ -37,8 +38,26 @@ func GetCommand(args []string) *exec.Cmd {
 		cmdPath = cmdPath + ".exe"
 	}
 
-	cmd := exec.Command(cmdPath, args...)
-	return cmd
+	fname, err := createTempCoverageFile()
+	if err != nil {
+		return exec.Command(cmdPath, args...)
+	}
+	args = append([]string{fmt.Sprintf("-test.coverprofile=%s", fname)}, args...)
+	return exec.Command(cmdPath, args...)
+}
+
+// createTempCoverageFile creates a coverage file for a CLI command under $TMPDIR.
```

Comment:
is `$TMPDIR` an env set by ioutil or something we set explicitly somewhere?

Reply:
I believe it's set by default [depending on your OS](https://en.wikipedia.org/wiki/TMPDIR). 
`ioutil` refers to  the [tempDir() function](https://github.com/golang/go/blob/master/src/os/file_unix.go#L340) in the end to figure out the location of your tmp directory.

For CodeBuild, I had to add the env variable TMPDIR that's set to /tmp. I added a new section in the README to set this variable locally if your environment doesn't set it.